.gitignore

```diff
@@ -10,6 +10,7 @@ LocalConfig.php
 *.css.map
 /build/
 /public/www.pid
+/.sass-cache/
```

Comment:
I think I had an old `.sass-cache` dir laying around and I added this so I didn't accidentally push it. I wasn't sure why the sass stuff all disappeared on me, but it looks like it was removed intentionally. I'll remove my local dir and this line.